web.py

```diff
@@ -139,5 +187,38 @@ def compile(emit, optimize, version, color, syntax):
         else:
             return {"result": split[1].decode()}
 
-os.chdir(sys.path[0])
-run(host='0.0.0.0', port=80, server='cherrypy')
+def main(args):
+    '''
+    -d: Debugging mode: turn on verbose debugging (and RUST_BACKTRACE=1)
+        Change the listening port to 8080 from the default 80.
+    -S: Skip playpen and direcly run rustc and the output binary.
+        To mitigate the risk, accepts only local connections by default.
```

Comment:
Gah, I meant multirust-rs, sorry.

You can generate all root directories by running init.sh, but you basically need to be running Arch Linux for that to work. I know, it's bad. I'd love the sandboxed playpen to run under other OSes as well, but I guess until that works we'll have to make do with what we have.